internal/presenters/presenter_sarif_results_pretty.go

```diff
@@ -84,22 +88,30 @@ Testing %s ...
 %s
 `,
 		meta.TestPath,
-		renderFindings(SortFindings(findings, summary.SeverityOrderAsc)),
+		renderFindings(SortFindings(findings, summary.SeverityOrderAsc), showIgnored, showOpen),
 		renderSummary(summary, meta),
 		getTip(),
```

Comment:
Issue: the output of `getTip()` needs to adapt based on the setting, see design.